node/src/types/deploy.rs

```diff
@@ -815,7 +859,12 @@ impl Deploy {
         ];
         let chain_name = String::from("casper-example");
 
-        let payment = rng.gen();
+        let payment = ExecutableDeployItem::ModuleBytes {
+            module_bytes: casper_types::bytesrepr::Bytes::new(),
```

Comment:
Let's import `casper_types::bytesrepr::Bytes`

Reply:
I moved it up to the imports under `cfg(test)`